server/operator.go

```diff
@@ -176,16 +182,19 @@ func (co *ChangePeerOperator) Do(region *metapb.Region, leader *metapb.Peer) (bo
 
 // TransferLeaderOperator is used to do leader transfer.
 type TransferLeaderOperator struct {
-	mustSuccess bool
+	count        int
+	maxWaitCount int
 
 	oldLeader *metapb.Peer
 	newLeader *metapb.Peer
 }
 
-func newTransferLeaderOperator(oldLeader, newLeader *metapb.Peer) *TransferLeaderOperator {
+func newTransferLeaderOperator(oldLeader, newLeader *metapb.Peer, waitCount int) *TransferLeaderOperator {
 	return &TransferLeaderOperator{
-		oldLeader: oldLeader,
-		newLeader: newLeader,
+		oldLeader:    oldLeader,
+		newLeader:    newLeader,
+		count:        0,
```

Comment:
Seems not, we must store the origin `maxWaitCount` to make sure the first time we return the transfer leader response. 
If we only use `count`, then we cannot distinct it.